darts/timeseries.py

```diff
@@ -1570,25 +1578,37 @@ def data_array(self, copy=True) -> xr.DataArray:
         """
         return self._xa.copy() if copy else self._xa
 
-    def pd_series(self, copy=True) -> pd.Series:
+    def to_series(
+        self,
+        copy: bool = True,
+        backend: ModuleType
+        | Implementation
+        | Literal["pandas", "polars", "pyarrow", "modin", "cudf"] = "pandas",
```

Comment:
using a literal means that we would have to keep track of what backends narwhals adds. Let's use a simple `str` instead (as they also do in their implementation). The curious user can read about it in the linked documentation

```suggestion
        backend: Union[ModuleType, Implementation, str] = "pandas",
```